.MAKE.sh

```diff
@@ -0,0 +1 @@
+./configure --prefix=/usr/local --sysconfdir=/etc/nut --with-modbus
```

Comment:
Am guessing this is temporary and you intend to remove it before making this not a draft.    I can certainly understand wanting that for your debugging, but it seems aligned to local preferences rather than something that should be in the source tree.